packages/core/src/indexing/database-index-api.ts

```diff
@@ -43,7 +43,7 @@ export interface IndexModelArgs {
 type IndexedData<DateType> = {
   stream_id: string
   controller_did: string
-  stream_content: any
+  stream_content: Record<string, any> | string
```

Comment:
Could this potentially be just a `Record`? Can be addressed in a different PR though.